Backpack/src/main/java/net/skyscanner/backpack/calendar/view/CalendarView.kt

```diff
@@ -81,9 +98,7 @@ internal class CalendarView constructor(context: Context, attr: AttributeSet?) :
     setBackgroundColor(ContextCompat.getColor(context, R.color.bpkWhite))
   }
 
-  /**
-   * Updates the title and selected month if the view has moved to a new month.
-   */
+  // TODO: Updates the title and selected month if the view has moved to a new month.
```

Comment:
Why did this turn into a TODO again?

Reply:
Because it doesn't actually do this right now, and is something the IOS version does